torchci/test/disableFlakyBot.test.ts

```diff
@@ -0,0 +1,200 @@
+            expect(JSON.stringify(body.body)).toContain("Platforms: ");
+            return true;
+        })
+        .reply(200, {});
+
+    await disableFlakyTestBot.handleFlakyTest(flakyTestA, [], octokit);
+
+    if (!nock.isDone()) {
+        console.error("pending mocks: %j", scope.pendingMocks());
+        console.error("pending mocks: %j", scope2.pendingMocks());
+    }
+  });
+
+  test("flaky test associated with an open issue should comment", async () => {
+    const scope = nock("https://api.github.com")
+        .post("/repos/pytorch/pytorch/issues/1/comments", (body) => {
+            const comment = JSON.stringify(body.body);
+            expect(comment).toContain("Another case of flakiness has been found");
+            expect(comment).toContain("Please verify");
+            return true;
+        })
+        .reply(200, {});
+
+    const issues = [{
+        number: 1,
+        title: "DISABLED test_a (__main__.suite_a)",
+        html_url: "https://api.github.com/repos/pytorch/pytorch/issues/1",
+        state: ("open" as "open" | "closed")
+    }];
+
+    await disableFlakyTestBot.handleFlakyTest(flakyTestA, issues, octokit);
+
+    if (!scope.isDone()) {
+        console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test("flaky test associated with a closed issue should reopen issue and comment", async () => {
+    const scope = nock("https://api.github.com")
+        .patch("/repos/pytorch/pytorch/issues/1", (body) => {
+            expect(body).toMatchObject({state: "open"});
+            return true;
+        })
+        .reply(200, {})
+        .post("/repos/pytorch/pytorch/issues/1/comments", (body) => {
+            const comment = JSON.stringify(body.body);
+            expect(comment).toContain("Another case of flakiness has been found");
+            expect(comment).toContain("Reopening");
+            return true;
+        })
+        .reply(200, {});
+
+    const issues = [{
+        number: 1,
+        title: "DISABLED test_a (__main__.suite_a)",
+        html_url: "https://api.github.com/pytorch/pytorch/issues/1",
+        state: ("closed" as "open" | "closed")
+    }];
+
+    await disableFlakyTestBot.handleFlakyTest(flakyTestA, issues, octokit);
+
+    if (!scope.isDone()) {
+        console.error("pending mocks: %j", scope.pendingMocks());
+    }
+    scope.done();
+  });
+});
+
+describe("Disable Flaky Test Bot Unit Tests", () => {
+    beforeEach(() => {
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+      jest.restoreAllMocks();
+    });
+
+
+    test("getTestOwnerLabels: owned test file should return proper module along with triaged", async () => {
+        const scope = nock("https://raw.githubusercontent.com/")
+            .get(`/pytorch/pytorch/master/test/${flakyTestA.file}.py`)
+            .reply(200, Buffer.from("# Owner(s): [\"module: fft\"]\nimport blah;\nrest of file"));
```

Comment:
nit: we can use `` and not have all the escapes and stuff. It might be annoying to edit this in the future.